Utilities/Create-AzureConfigFilesForJenkins.ps1

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+<#
+.SYNOPSIS
+    This file creates 'Property Files' for jenkins dynamic menu creation.
+    These 'Property Files' can be used to create dynamic jenkins menus using parameter type - "Extended Choice Parameter".
+
+.PARAMETER
+	See source code for the detailed parameters
+
+.NOTES
+	PREREQUISITES:
+	1) You should have a valid Azure subscription and read access.
+
+.EXAMPLE
+	Example 1 :
+    .\Utilities\Create-AzureConfigFilesForJenkins.ps1
+	Example 2 :
+    .\Utilities\Create-AzureConfigFilesForJenkins.ps1 -SecretFilePath .\AzureSecretFile.xml `
+        -ImageLocation "northeurope" `
+        -ImagePublishers "Canonical,SUSE"
+#>
+
+param (
+    [string] $SecretFilePath = "",
+    [string] $ImageLocation = "westus2",
+    [string] $ImagePublishers = "Canonical,SUSE,Oracle,CoreOS,RedHat,OpenLogic,credativ,kali-linux,clear-linux-project,MicrosoftOSTC,MicrosoftRServer,MicrosoftSharePoint,MicrosoftSQLServer,MicrosoftVisualStudio,MicrosoftWindowsServer,MicrosoftWindowsServerEssentials,MicrosoftWindowsServerHPCPack,MicrosoftWindowsServerRemoteDesktop"
```

Comment:
can you fetch this directly from Azure if no custom param value was given?
The list of publishers is stable let's say, but it can change in the future.

If no improvement can be made to this design, please at least split into multiple lines.